test-requirements.txt

```diff
@@ -0,0 +1,4 @@
+pytest
```

Comment:
You may want to explicitly define the version of your test dependencies so when you update them, you have to do this manually and you know its not the dependencies that fail your tests but the tests themselves.

Reply:
Done